dataservice/api/README.md

```diff
@@ -1,4 +1,58 @@
-Primary API for interacting with the Kid's First Data Model.
+<center>
+![Kids First](/logo)
+</center>
+
+Welcome to the developer documentation for the Kids First Dataservice API!
+
+The dataservice is the primary hub for retrieval and creation of Kids First
+data. It exposes functionality using a familiar REST interface over HTTP
+requests.
+
+# Connect to the API
+
+The API is currently only available internally inside the Kids First cloud
+environments.
+
+## curl
+
+Curl is the quickest way to communicate with the api from a unix shell
+
+```bash
+curl -H "Content-Type: application/json" http://kf-api-dataservice-qa.kids-first.io/status
+```
+
+To give input to the API:
+```bash
+curl -H "Content-Type: application/json" http://kf-api-dataservice-qa.kids-first.io/status -d '{ "external_id": "my identifier" }'
+```
+
+## Python
+
+We suggest using the popular `requests` package to interact with the API.
+
+```python
+import requests
+
+resp = requests.get('http://kf-api-dataservice-qa.kids-first.io/status',
+                    headers={'Content-Type': 'application/json'})
+
+print(resp.json())
+```
+
+To give input to the API:
+```python
+import requests
+
+body = {
+  "external_id": "my identifier"
+}
+
+resp = requests.post('http://kf-api-dataservice-qa.kids-first.io/participants',
```

Comment:
Following along with the above example, maybe change to `/studies`. Another reason to use studies endpoint is study entity is pretty simple and `external_id`  is the only required field right now so the POST studies example will work whereas this POST participants will break right now